test/mpi/mpi_util.h

```diff
@@ -34,6 +34,7 @@ static inline MPI_Datatype ucc_dt_to_mpi(ucc_datatype_t dt) {
     case UCC_DT_FLOAT16:
     case UCC_DT_INT128:
     case UCC_DT_UINT128:
+    case UCC_DT_BFLOAT16:
```

Comment:
@lappazos plz advice you you are running ucc_test_mpi with bfloat16/float16? Why i try to run it, it get "Unsupported dt".